hbase-common/src/main/java/org/apache/hadoop/hbase/util/DNS.java

```diff
@@ -35,13 +35,21 @@
   // the specification of server hostname is optional. The hostname should be resolvable from
   // both master and region server
   @InterfaceAudience.LimitedPrivate(HBaseInterfaceAudience.CONFIG)
-  public static final String RS_HOSTNAME_KEY = "hbase.regionserver.hostname";
+  public static final String UNSAFE_RS_HOSTNAME_KEY = "hbase.unsafe.regionserver.hostname";
```

Comment:
I am not sure if Jira mentioned about this config to be renamed as `unsafe`.

Reply:
Even I missed it initially, it is mentioned in the comment.

Reply:
I see, got it, just saw the comment. Thanks